spec/Appendix D -- Conformance & Licensing.md

```diff
@@ -0,0 +1,64 @@
+# D. Appendix: Conformance & Licensing
```

Comment:
Does it make sense to have these be in the same appendix? I feel like having an extra focussed appendix is not too costly, and "conformance" and "licensing" feel somewhat separate to me.